# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libfwbuilder/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur libfwbuilder-2.0.10-orig/src/fwbuilder/dns.h libfwbuilder-2.0.10/src/fwbuilder/dns.h
--- libfwbuilder-2.0.10-orig/src/fwbuilder/dns.h	2005-03-30 07:50:13.000000000 +0000
+++ libfwbuilder-2.0.10/src/fwbuilder/dns.h	2006-04-03 14:22:56.862873000 +0000
@@ -92,7 +92,7 @@
      */
     static void init();
     
-    DNS::DNS();
+    DNS();
 
     /**
      * Finds IP adddresses of the host with given host name.
